sdk/core/azure-core/test/ut/transport_adapter_implementation_test.cpp

```diff
@@ -54,7 +55,10 @@ namespace Azure { namespace Core { namespace Test {
       TransportAdapter,
       testing::Values(
           GetTransportOptions("winHttp", std::make_shared<Azure::Core::Http::WinHttpTransport>()),
-          GetTransportOptions("libCurl", std::make_shared<Azure::Core::Http::CurlTransport>())),
+          GetTransportOptions("libCurl", std::make_shared<Azure::Core::Http::CurlTransport>()),
+          GetTransportOptions(
+              "staticLibCurl",
+              std::make_shared<Azure::Core::Http::StaticCurlTransport>())),
```

Comment:
Since this test failure is intermittent, I will leave it up to you, Victor, to investigate the root cause and take action as part of:
https://github.com/Azure/azure-sdk-for-cpp/issues/3065